src/ASM/LR/ASMu3Dmx.C

```diff
@@ -369,9 +369,11 @@ bool ASMu3Dmx::integrate (Integrand& integrand,
 
   bool ok = true;
   for (size_t t = 0; t < group.size() && ok; t++)
-//#pragma omp parallel for schedule(static)
+#pragma omp parallel for schedule(static)
     for (size_t e = 0; e < group[t].size(); e++)
     {
+      if (!ok)
+        continue;
```

Comment:
Is this necessary?

Reply:
As we cannot break out of the innermost loop, we need this to halt further work if an error has been flagged. Same reason as the continue difference below.

Reply:
OK, I got it. It's fine then.